daprdocs/content/en/reference/components-reference/supported-bindings/storagequeues.md

```diff
@@ -47,6 +49,7 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | queue | Y | Input/Output | The name of the Azure Storage queue | `"myqueue"` |
 | ttlInSeconds | N | Output | Parameter to set the default message time to live. If this parameter is omitted, messages will expire after 10 minutes. See [also](#specifying-a-ttl-per-message) | `"60"` |
 | decodeBase64 | N | Output | Configuration to decode base64 file content before saving to Blob Storage. (In case of saving a file with binary content). `true` is the only allowed positive value. Other positive variations like `"True", "1"` are not acceptable. Defaults to `false` | `true`, `false` |
+| queueEndpoint | N | Input/Output | Optional custom endpoint URL. This is useful, for example, with "[production-style URLs](https://github.com/Azure/azurite#production-style-url)" when using the [Storage emulator](https://github.com/Azure/azurite) | `"https://accountName.queue.example.com:10001"` |
```

Comment:
Is this actually useful without the storage emulator? I couldn't find any example. If it is limited to Azurite / emulators I think we should make it more clear like.. `Optional. Only applies to use with Storage Emulator.`

Reply:
@rzyns - Can you reply to this question from @berndverst?

Reply:
Oh, sorry @berndverst, I didn't notice you had posted a comment on this issue--I was mostly watching for activity on the main PR. Anyway, this isn't only useful for the storage emulator, it also be useful when you have a storage endpoint [mapped to a custom domain](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-custom-domain-name). I haven't tried that with storage queues yet and so I haven't explicitly tested it, but I wouldn't say it's "only  useful with the storage emulator." It's also useful for integration testing, or any scenario in which you need to direct the API traffic to a specific endpoint.

Reply:
I suggest that you add other cases where this queueEndpoint can be used, as you described. 

Also ideally this property should IMO be called "queueEndpointUrl" is this is a URL.

Reply:
I agree about the field name, @msfussell, and I've committed the change. I'm reluctant to add other use cases to the documentation, though, since I've only explicitly tested the storage emulator case. If I do get a chance to actually test out the custom domain functionality later, I'd be happy to submit another PR

Reply:
@rzyns - I would still mention these are scenarios, and feel to add a caveat that these have not been tested. That way, you may get someone else to try this out and give a response.